### DISCUSSION TITLE
other/2022/07/16/change-article-comments/

### DISCUSSION THREAD

**Discussion by giscus** [Moosphan/Moosphan.github.io#6] [Comments]
# other/2022/07/17/change-article-comments/

修技术之道，日三省吾身 | App Dev & Freelance Thinker，Hacker Growth | 这里是 @道轲(Dorck) 的个人博客，与你一起探寻世界。

**Comment by Moosphan** [comment]
See how to avoid rendering html in markdown code blocks:
> https://github.com/jekyll/jekyll/issues/9114

**Comment by tujiaw** [comment]
我也试试giscus
